articles/libraries/auth0js/v9/migration-guide.md

```diff
@@ -30,7 +30,7 @@ Migration to v9 will depend on how you are using auth0.js:
 | In your application, to implement embedded login | Required |
 | In a customized hosted page | Not Supported |
 
-(*) There a certain usage patterns with centralized login and auth0.js v8 that don't require migration. However, given that for those scenarios just updating the library version will work, we recommend that you always migrate to v9.
+(*) There a certain usage patterns with centralized login and auth0.js v8 that don't require migration. However, given that for those scenarios just updating the library version will work, we recommend that you always migrate to v9 when embedding auth0.js.
 
```

Comment:
What does it mean to 'embed auth0.js'? If you use auth0.js v8 for centralized login in your website, you should upgrade to v9

Reply:
Just noting that we are not recommending that you upgrade if you are using it inside a hosted page? I can reword to make it clearer.